persistence/src/main/java/de/cyface/persistence/PersistenceLayer.java

```diff
@@ -534,7 +547,7 @@ public void deletePoint3dData(final long measurementIdentifier) {
         Cursor geoLocationCursor = null;
         Cursor eventCursor = null;
         try {
-            eventCursor = loadEvents(measurementIdentifier);
```

Comment:
had to rename it or else it clashes with the

public List<Event> loadEvents(mid) method introduced. But the former method was anyway private.